src/sync_dashboards/main.py

```diff
@@ -48,7 +48,8 @@ def get_all_linked_dashboards(sdk: methods.Looker31SDK) -> dict:
     """
     remote_config = {}
 
-    dashboards = sdk.search_dashboards(deleted=False)
+    transport_options = looker_sdk.rtl.transport.TransportOptions({"timeout": 60 * 5})
```

Comment:
Do you think we can test in a staging env? Increasing timeouts may affect other processes.